src/features/user/ManageProjects/components/ProjectSites.tsx

```diff
@@ -620,7 +421,7 @@ export default function ProjectSites({
           </Button>
 
           <Button
-            onClick={handleSubmit(uploadProjectSiteNext)}
+            onClick={handleSubmit(uploadProjectSite)}
             variant="contained"
```

Comment:
After replacing uploadProjectSiteNext with uploadProjectSite, we are moving to the next tab after adding any site, while we might want to stay on the tab